packages/protocol/migrationsConfig.js

```diff
@@ -13,7 +13,7 @@ const DefaultConfig = {
     selectIssuersWaitBlocks: 4,
   },
   blockchainParameters: {
-    gasForNonGoldCurrencies: 134000,
+    gasForNonGoldCurrencies: 166000,
```

Comment:
Why 32000 more?

Reply:
To match the default set as a part of https://github.com/celo-org/celo-blockchain/pull/602. The payment of the gateway fee adds an additional credit operation, which we have been estimating at 32000 gas.

Reply:
I don't think this is exactly correct, but only because I don't believe what we've been doing is exactly correct. So not in the scope of this PR

Reply:
Ya I was wondering why we don't actually meter the ERC20 calls and either charge only what is used, or keep the intrinsic gas, then issue a refund if its under. Is there a reason for that?